app/Repositories/SignupRepository.php

```diff
@@ -43,6 +45,10 @@ public function create($data)
             $signup->save(['timestamps' => false]);
         }
 
+        // Now that the signup exists, set the signup_id on the signup event
```

Comment:
@sbsmith86 agreed, I was trying to figure out a better way to do it too and was blanking, but I haven't slept on it yet